Documentation/AccessControl/Index.rst

```diff
@@ -16,11 +16,38 @@ period, all the way through to requiring visitors to log in to view content.
 
 These various options are presented in this chapter.
 
+.. container:: row m-0 p-0
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Visibility<visibility>`
+
+         .. container:: card-body
+
+            When an element is hidden, be it a page, a content element or a news item,
+            it will not display in the frontend. You can, however, preview it in the backend.
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Frontend Login<frontend-login>`
+
+         .. container:: card-body
+
+            You can specify a period of time for content to be visible.
```

Comment:
The text does not match the header?

Reply:
Correct - this should be resolved now. Thank you